src/transformers/models/data2vec/modeling_data2vec_text.py

```diff
@@ -400,14 +400,14 @@ def __init__(self, config):
 
     def forward(
         self,
-        hidden_states,
-        attention_mask=None,
-        head_mask=None,
-        encoder_hidden_states=None,
-        encoder_attention_mask=None,
-        past_key_value=None,
-        output_attentions=False,
-    ):
+        hidden_states: torch.Tensor,
+        attention_mask: Optional[torch.FloatTensor] = None,
+        head_mask: Optional[torch.FloatTensor] = None,
+        encoder_hidden_states: Optional[torch.FloatTensor] = None,
+        encoder_attention_mask: Optional[torch.FloatTensor] = None,
+        past_key_value: Tuple[Tuple[torch.FloatTensor]] = None,
+        output_attentions: Optional[bool] = False,
+    ) -> Union[Tuple]:
```

Comment:
These outputs just look like Tuples of torch.Tensors to me? I may just remove the Union all together

Reply:
I'll try modifying imports on dependent models as well